app/views/two_factor_authentication/totp_verification/show.html.slim

```diff
@@ -14,6 +13,7 @@ p.mt-tiny.mb0#code-instructs = @presenter.help_text
       'aria-describedby': 'code-instructs', maxlength: Devise.otp_length, autocomplete: 'off'
   = submit_tag 'Submit', class: 'btn btn-primary align-top'
 
+p.mt3.mb1#code-instructs = @presenter.help_text
```

Comment:
Same here, once the div around the fallback links is gone, just switch this to `p.mt3#code-instructs`.